inlang/source-code/create-project/src/tryAutoGenProjectConfig.ts

```diff
@@ -78,20 +77,26 @@ export async function tryAutoGenModuleConfig(args: {
 		)
 	}
 
-	args.baseConfig.modules = [pluginUrls[pluginName]!, ...standardLintRules]
+	const config: ProjectConfig = {
+		$schema: "https://inlang.com/schema/project-config",
+		sourceLanguageTag: "en",
+		languageTags: ["en"],
+		modules: [pluginUrls[pluginName]!, ...standardLintRules],
+		settings: {},
+	}
 
 	const pluginId: Plugin["meta"]["id"] = `plugin.inlang.${pluginName}`
 
-	args.baseConfig.settings = {
+	config.settings = {
 		[pluginId]: { pathPattern },
-		...args.baseConfig.settings,
 	} satisfies ProjectConfig["settings"]
```

Comment:
Why arent you creating the config in one go? Is there a reason to use `config.settings = `?